src/databricks/sql/client.py

```diff
@@ -501,6 +505,11 @@ def execute(
         """
         if parameters is None:
             parameters = []
+        elif (
+            self.connection.get_session_protocol_version()
+            < ttypes.TProtocolVersion.SPARK_CLI_SERVICE_PROTOCOL_V8
+        ):
+            raise Error("Parameterized operations are not supported by this server")
```

Comment:
Can we include the protocol version in the exception? Potentially with a reference to the earliest version of DBR that should support parameters? Just something to help users discern what they can do to resolve this without digging into the source code.

Reply:
Case in point: a user of the node driver ran into this and opened an issue https://github.com/databricks/databricks-sql-nodejs/issues/184

It would be nice to communicate the more about the nature of this across all our drivers.

Reply:
fair enough. I can edit them to note that DBR version 14.1 is the minimum version.

Reply:
@nithinkdb can we do the same for Nodejs driver?

Reply:
Yep.